docs/glossary.md

```diff
@@ -0,0 +1,25 @@
+(_proxy_py_glossary)=
+# {{ project }} Glossary
```

Comment:
@abhinavsingh I declared this variable substitution so you can use it as `{{ project }}` in Markdown documents and as `|project|` in RST.